core/modules/views_ui/js/views-admin.js

```diff
@@ -197,7 +197,7 @@ Backdrop.viewsUi.AddItemForm = function ($form) {
 
 Backdrop.viewsUi.AddItemForm.prototype.handleCheck = function (event) {
   var $target = $(event.target);
-  var label = $target.next().text().trim();
+  var label = $target.closest('td').next().text().trim()
```

Comment:
I resolved a conflict with this line, but dropped a semi-colon:

```suggestion
  var label = $target.closest('td').next().text().trim();
```